jerry-debugger/src/lib/__tests__/protocol-handler.test.ts

```diff
@@ -68,19 +95,78 @@ describe('onSourceCode', () => {
   });
 
   it('does not call scriptParsed after only SOURCE message', () => {
-    const array = Uint8Array.from([SP.JERRY_DEBUGGER_SOURCE_CODE,
-      'a'.charCodeAt(0), 'b'.charCodeAt(0), 'c'.charCodeAt(0)]);
+    let array = encodeArray(SP.JERRY_DEBUGGER_SOURCE_CODE, 'abc');
+    // code = 'abc'
     handler.onSourceCode(array);
     expect(delegate.onScriptParsed).toHaveBeenCalledTimes(0);
   });
 
   it('immediately calls scriptParsed from END message', () => {
-    const array = Uint8Array.from([SP.JERRY_DEBUGGER_SOURCE_CODE_END,
-      'a'.charCodeAt(0), 'b'.charCodeAt(0), 'c'.charCodeAt(0)]);
+    let array = encodeArray(SP.JERRY_DEBUGGER_SOURCE_CODE_END, 'abc');
```

Comment:
why `let`?

Reply:
Just a copy/paste thing after revising the form of those a few times for succinctness.